### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position

### ISSUE THREAD

**Issue by Muchmorepig** [lmdu/pyfastx#56]
![image](https://user-images.githubusercontent.com/68462246/203269415-a5d0f6fb-c795-4639-8b28-babd040bd097.png)
python version: 3.8.12
pyfastx version: 0.8.4

```python
import pyfastx 
fq2 = pyfastx.Fastq("./tmp/tmp2_2.fq.gz")
for read in fq2:
    print(read.seq)
```
here is the header of my fq file:
```
@A01225:536:HFNG7DSX5:3:1101:1217:1000 2:N:0:TAAGGCGA+GCGATCTA
CAAGCGTTGGCTTCTCGCATCTGTACGGTGTCAACGCTTAATCCACGTGCTTGAGAGGCCAGAGCATTCGACACAGAATTTTTTTTTTTTTTCGGTTAGTACGAGATGGCTTCACACCTCGCGGTGCATCCAACTCTCGCCTATCGAAGT
+
,FFFFFF:FFFFFFFFFFFFFFFFFFFFFFF:FF:FFFF:,FFFFFFFFFFFFFFFFFFFFFFFFF:FFFFFFFFFFFF:FFFFFFFFFFFF:F,FF::FFF:FF,:FF,:F,FFFFF,FFF:FFFF,FFFF,F,FFF:,F:FFF,FF:F
@A01225:536:HFNG7DSX5:3:1101:1253:1000 2:N:0:TAAGGCGA+GCGATCTA
CAAGCGTTGGCTTCTCGCATCTTCGTCCATCATACTAGTAATCAACGTGCTTGAGAGGCCAGAGCATTCGAAGATCTTTTTTTTATTTTTTTTTGTTTTTGTTTTAAAAAATTTATGAAAAAATTGATACGCCAATTATTACTAAAACTC
+
:F:,F,FFFFFF,FFFFFFFFFF:,FF:FFFF,F,FFFFFFFFF,FFFFFFFFFFFF,F,FFF,FFFFF,F,F,,,:FFFFFFF,F:F:FFFF:,F:,FF,,::F:,FFFF,FF,,,,F,FF,F:::,:,,F,:F,:F,F:,:F:,F,F,
```

**Comment by Muchmorepig** [comment]
can anyone help me ?

**Comment by lmdu** [comment]
I have test pyfastx with fastq sequence you have provided, I did not suffer from the same. If it's convenient, would you please send me the whole test file. Thank you for reporting this issue, I found another bug in Fastq parser. By the way, your running environment does not seem to like terminal.

**Comment by lmdu** [comment]
Fixed in verions of 0.9.1.

**Comment by Muchmorepig** [comment]
thanks
